images/base/Dockerfile

```diff
@@ -140,7 +140,7 @@ RUN git clone --filter=tree:0 "${CONTAINERD_CLONE_URL}" /containerd \
 # stage for building runc
 FROM go-build AS build-runc
 ARG TARGETARCH GO_VERSION
-ARG RUNC_VERSION="v1.1.13"
+ARG RUNC_VERSION="v1.2.2"
```

Comment:
```suggestion
ARG RUNC_VERSION="v1.2.3"
```

?

Reply:
I intend to. this is still WIP, been following up on this nightmare: with unknown automation accessing k8s-ci-robot (which is owner/admin of all the github orgs) 🙃 https://kubernetes.slack.com/archives/C09QZ4DQB/p1733947637418569